<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="load_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);
	border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);
	border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>인식할 사진을 불러와주세요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);
	border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>이미지 전송</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>481</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>80</y>
     <width>510</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../ui_img/title.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>load_btn</zorder>
  <zorder>exit_btn</zorder>
  <zorder>path_label</zorder>
  <zorder>send_btn</zorder>
  <zorder>pic_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
